source/Agents/AgentFactory.hpp

```diff
@@ -19,53 +21,53 @@
 
 namespace walle {
 
-// Forward-declare since it's easier to understand how AddXAgent works when the structs are defined nearby
+// Forward-declare since it's easier to understand how AddXAgent works 
+// when the structs are defined nearby
 struct PacingAgentData;
 struct PathAgentData;
 struct AStarAgentData;
 struct TrackingAgentData;
 
 class AgentFactory {
- private:
-  cse491::WorldBase &world;
+private:
+    cse491::WorldBase &world;
```

Comment:
A brief description (one line) would be good for completeness, or anything that we have defined ourselves.